pyanaconda/modules/common/structures/subscription.py

```diff
@@ -533,3 +555,41 @@ def consumed_entitlement_count(self) -> Int:
     @consumed_entitlement_count.setter
     def consumed_entitlement_count(self, consumed_entitlement_count: Int):
         self._consumed_entitlement_count = consumed_entitlement_count
+
+
+class OrganizationData(DBusData):
+    """Data about a single organization in the Red Hat account system.
+    A Red Hat account is expected to be member of an organization,
+    with some accounts being members of more than one organization.
+    """
+
+    def __init__(self):
+        self._id = ""
+        self._name = ""
+        self._simple_content_access_enabled = ""
```

Comment:
I would expect None or False? Also not accessible via property...

Reply:
Good catch - this was a leftover from the original & currently stalled PR for Simple Content Access handling. We can drop it for now & add it back in some for when/if we revisit the SCA handling PR.